Content.Client/UserInterface/GhostGui.cs

```diff
@@ -7,7 +7,8 @@ namespace Content.Client.UserInterface
 {
     public class GhostGui : Control
     {
-        public Button ReturnToBody = new Button(){Text = "Return to body"};
+        public Button ReturnToBody = new Button() {Text = "Return to body"};
+        public Button ReturnToCloneBody = new Button() {Text = "Return to cloned Body"};
```

Comment:
These two strings need to be wrapped with `Loc.GetString()`

Reply:
Done.